ctrlf.el

```diff
@@ -129,6 +136,30 @@ I have literally no idea why this is needed.")
 Assume that S2 has the same properties throughout."
   (apply #'propertize s1 (text-properties-at 0 s2)))
 
+(defun ctrlf--regexp-translate (str)
+  "Translate the user inputted regular expression STR.
+A single space character is translated into \".*\", while N spaces (N >= 2) are
+translated in to N-1 spaces. Space or spaces escaped with \"\\\" will not be
+translated."
```

Comment:
This seems sketchy to me. What if I have an escaped backslash followed by a space? Your code will think that the space is being escaped, even when it isn't. But this is a corner case that I don't care about too much, so it would be fine if you just include a comment documenting it.

Reply:
Yes, that's a problem. In fact I also found some other cases where my translator fails. I've fixed all them, and I'll add comments to explain.